### ISSUE TITLE
Start and stop logging button

### ISSUE THREAD

**Issue by wmaciel** [ponewheel/android-ponewheel#42]
If I am not mistaken, as it stands now, the app starts logging the board's info as soon as it connects.

If we had a start and stop logging button, similar to the ride mapping in the original app, it would make easier to know when a ride starts and stops, making it easier to know when to keep polling gps data and how to split the logs in the database.

**Comment by ebabel** [comment]
@wmaciel To be honest I like not having to manage logging as a user. I never want to forget to start it and then crash and wish I had started it. Also adding the button could add clutter to the UI.

I would even prefer not to manage the connection. Just auto connect when the app starts like the official app does.

You are right about the app starting logging on connection but also note that if it disconnects and reconnects within one minute then the same ride will resume. Does that fix what you didn't like about it or was there something more?

**Comment by wmaciel** [comment]
@ebabel I see your point and understand it.

I think what I am actually missing is some kind of feedback from the app to assure me that it is actually logging. Something other than the momentary toast message. Maybe a blinking red "light" somewhere on the ui. idk.

Throwing ideas around.

**Comment by ebabel** [comment]
Yeah, I like this idea. Maybe add in a new 'card' below the ride mode.
Maybe take the full width and do something with the right side, like:
Logging: ON   Location: ON

Or have a saved rides counter? Just a thought

**Comment by ebabel** [comment]
Or we could shrink the lights card and put it next to that.
I'm imagining a smaller lights card could show two light bars and be a button that goes into a screen for just the lights?

**Comment by ebabel** [comment]
Hmm, maybe a little nicer to also have a ready state?
"Logging: READY" when not actively riding, and "Logging: ON"

**Comment by kwatkins** [comment]
Hmm - the trip connected time in the device attributes list flips on when the board is connected. We could add under that the "Logging" on/off state, basically another device status state. That would be the easiest thing to do. Wouldn't have purty colors tho (not sure how easy it is to add color to the text output w/ the data binding mess). From there it'll be available & then explore making it fancy in the UI. Hmm... It also might be nice to add it to the toolbar, an icon that lights off/on and when touched opens up logged ride activities or something.
